commands/download_mta_op_logs_command.go

```diff
@@ -145,7 +171,44 @@ func createDownloadDirectory(downloadDirName string) (string, error) {
 	return filepath.Abs(filepath.Dir(downloadDirName))
 }
 
+func sanitizeDirectoryName(downloadDir string) string {
+	// Check if directory name ends with the os specific path separator
+	if !strings.HasSuffix(downloadDir, string(os.PathSeparator)) {
+		//If there is no os specific path separator, put it at the end of the directory name
+		downloadDir = downloadDir + string(os.PathSeparator)
+	}
+	// Check if directory name starts with the os specific path separator and trim it
+	downloadDir = strings.TrimPrefix(downloadDir, string(os.PathSeparator))
+	return downloadDir
+}
+
 func saveLogContent(downloadDir, logID string, content *string) error {
 	ui.Say("  %s", logID)
 	return ioutil.WriteFile(downloadDir+"/"+logID, []byte(*content), 0644)
 }
+
+type dmolCommandFlagsValidator struct{}
+
+func (dmolCommandFlagsValidator) ValidateParsedFlags(flags *flag.FlagSet) error {
+	if hasValue(flags, "i") && hasValue(flags, "mta-id") {
+		return fmt.Errorf("Option -i and option --mta-id are incompatible")
+	}
+	hasMtaId := hasMtaId(flags)
+	return NewDefaultCommandFlagsValidator(map[string]bool{"i": !hasMtaId, "mta-id": hasMtaId}).ValidateParsedFlags(flags)
```

Comment:
I understand this but in future if we add another dependendant flag to -i, we MUST NOT forget to change this !hasMtaId. So, please, use the hasValue(flags, "i")

Reply:
ok

Reply:
Is this changed?